packages/calypso-e2e/src/lib/flows/start-import-flow.ts

```diff
@@ -209,14 +226,33 @@ export class StartImportFlow {
 	 * @param {string} siteSlug The site slug URL.
 	 */
 	async startSetup( siteSlug: string ): Promise< void > {
-		const route =
-			this.framework === 'signup' ? '/start/setup-site/intent' : '/setup/site-setup/intent';
+		const route = '/setup/site-setup/intent';
```

Comment:
Remember last time we were talking about the intent step which is no longer using anymore. Reference here: p1699330829163449/1698678260.781409-slack-C01A60HCGUA

Should we remove this instead?

Reply:
Yes. We can do it, but I'm suggesting to be in the next PR.

I've created a ticket: https://github.com/Automattic/wp-calypso/issues/84503